helm-charts/charts/kube-starrocks/templates/starrocks/starrockscluster.yaml

```diff
@@ -14,9 +14,11 @@ spec:
   starRocksFeSpec:
     image: "{{ .Values.starrocksFESpec.image.repository }}:{{ .Values.starrocksFESpec.image.tag }}"
     replicas: {{ .Values.starrocksFESpec.replicas }}
-{{- if .Values.starrocksFESpec.resources }}
-{{ toYaml .Values.starrocksFESpec.resources | indent 4 }}
-{{- end }}
+    {{- if .Values.starrocksFESpec.resources }}
+    {{- toYaml .Values.starrocksFESpec.resources | nindent 4 }}
+    {{- else if .Values.starrocksFESpec.resource }}
```

Comment:
does helm chart template support comments? shall add comments here to explain why if-else if here.